src/blockchain_utilities/bootstrap_serialization.h

```diff
@@ -78,7 +95,7 @@ namespace cryptonote
         FIELD(block)
         FIELD(txs)
         VARINT_FIELD(block_weight)
-        VARINT_FIELD(cumulative_difficulty)
+        FIELD(cumulative_difficulty)
```

Comment:
What's the rationale for not using VARINT_FIELD?

Reply:
The new difficulty type is not a POD type. I suppose I could theoretically add some template functions to make things work out, but it looks like a lot of time spent fiddling with things with no 100% guarantee it can be done sanely.